tutorial/markdown/nodejs/quickstart-typescript/quickstart-typescript.md

```diff
@@ -0,0 +1,483 @@
+---
+# frontmatter
+path: "/tutorial-quickstart-nodejs"
+# title and description do not need to be added to markdown, start with H2 (##)
+title: Using Typescript with Node.js & Express
+short_title: Node.js and Express
+description:
+  - Build a basic REST API using Type Script, Express and the Couchbase Node.js SDK
+  - See how you can fetch data from Couchbase using SQL++ queries
+  - Explore CRUD operations in action with Couchbase
+content_type: quickstart
+filter: sdk
+technology:
+  - kv
+  - query
+tags:
+  - Express
+  - REST API
+sdk_language: 
+  - nodejs
+length: 30 Mins
+---
+
+In this tutorial, you will learn how to connect to a Couchbase Capella cluster to create, read, update, and delete documents and how to write simple parametrized SQL++ queries.
+
+
+## Prerequisites
+
+To run this prebuilt project, you will need:
+
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/nodejs-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+  - To run this tutorial using a self managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- [LTS Node.js Version](https://nodejs.org/en/download)
+- Loading Travel Sample Bucket
+  If travel-sample is not loaded in your Capella cluster, you can load it by following the instructions for your Capella Cluster:
+
+  - [Load travel-sample bucket in Couchbase Capella](https://docs.couchbase.com/cloud/clusters/data-service/import-data-documents.html#import-sample-data)
+
+> Note that this tutorial is designed to work with the latest Node SDK version (4.x) for Couchbase. It will not work with the older Node js versions  for Couchbase without adapting the code.
```

Comment:
Node js -> Node.js
Also extra space before for Couchbase

Reply:
Updated and removed extra space.